src/main/java/com/example/kadai10/mapper/MovieMapper.java

```diff
@@ -0,0 +1,25 @@
+package com.example.kadai10.mapper;
+
+import com.example.kadai10.entity.Movie;
+import com.example.kadai10.form.MovieForm;
+import org.apache.ibatis.annotations.Insert;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Options;
+import org.apache.ibatis.annotations.Select;
+
+import java.util.List;
+import java.util.Optional;
+
+@Mapper
+public interface MovieMapper {
+
+    @Select("SELECT * from movies")
```

Comment:
細かいですが、SQLの予約語は大文字に統一しましょう。
```suggestion
    @Select("SELECT * FROM movies")
```

Reply:
大文字に修正しました！

Reply:
問題ないです！
１点だけ覚えておいてほしいことがあります。

[#1指摘対応](https://github.com/isozaki29/kadai10/pull/1/commits/6dff8f3e34a5fc8ecbbca43f8b2cce06db10e61c)
のコミットメッセージですが、なるだけなにを修正したのか、もっというとなぜ修正したのかがわかるようなコミットメッセージがよいです。

「指摘対応」だと運用していてバグがあったときにコミットログをさかのぼってもどこでなにが行われたのかわかりづらく問題の原因特定を遅延させることに繋がります。

なるだけコミットは修正の単位を明確にしてわけるのがbetterです。

Reply:
コメントありがとうございます！
コミットメッセージについて承知しました。